circle.yml

```diff
@@ -0,0 +1,5 @@
+test:
+  post:
+    - bundle exec rspec
+    - bundle exec rubocop
+    - bundle exec reek .
```

Comment:
It's better to run linters first. They can catch silly mistakes before the testing which usually takes a lot of time. Consider:
```
bundle exec rubocop
bundle exec reek .
bundle exec rspec
```